examples/reprojection-by-code.html

```diff
@@ -15,7 +15,7 @@
   <span class="col-auto">
     <span class="input-group">
       <label class="input-group-text" for="epsg-query">Search projection:&nbsp</label>
-      <input type="text" id="epsg-query" placeholder="4326, 27700, 3031, US National Atlas, Swiss, France, ..." class="form-control" size="50" />
+      <input type="text" id="epsg-query" placeholder="4326, 27700, 3031, ..." class="form-control" size="50" />
```

Comment:
Maybe replace names with the codes 9311, 2154, 21781
More significantly 9311 does not work with the definition returned by the new lookup.

Reply:
@mike-000 What do you mean by

> 9311 does not work

I tried and the result looks correct to me.